lib/assets/javascripts/twine.js.coffee

```diff
@@ -326,5 +336,8 @@ setupEventBinding = (eventName) ->
     return teardown: ->
       $(node).off eventName, onEventHandler
 
+  Twine.bindingTypes["data-bind-event-#{eventName}"] = Twine.bindingTypes["bind-event-#{eventName}"]
```

Comment:
Probably cleaner to have a line at the end that does something like
```coffeescript
for key in Object.keys(Twine.bindingTypes)
  Twine.bindingTypes["data-#{key}"] = Twine.bindingTypes[key]
```

And then have a test that ensures that for every `bind-whatever